### ISSUE TITLE
Air system seems to have some issues

### ISSUE THREAD

**Issue by Dinerell** [mcenderdragon/Futurepack-API#970]
The Life Support System works like normal and looses oxygen but the E-Scanner shows different values on each scan sometimes dropping down back from about 25 min duration by having 4-5% at the end in a completly closed up room of 36 blocks OR it doesn't loose oxygen and the room is supposed to have about 1h left but 0,1% filled in a room of 183 blocks OR it does loose air but the filled ratio stays at 0,0%. There is also definitely no air because I take damage in survival. I used an oxygen tank right before this happened and it just stays this way (even after an restart).

1. Build a enclosed room out of Metal Blocks (suitable for spaceships).
2. Place a water pool of 3x3 blocks.
3. Place a Fluid Intake in the middle of the pool
4. Place down some sort of power generator and cable (like Solar Panels)
5. Place a Life Support System on the Fluid Intake
6. Scan some blocks
7. Taking air via an oxygen tank
8. Look how the Oxygen Tank is filled
9. It never refills again in this (game) world in any dimension without breathable environment (despite being not used for a while) and how you take damage without it

**Screenshots**
![2023-05-26_19 37 21](https://github.com/mcenderdragon/Futurepack-API/assets/33810746/8213abed-0c5a-4bb7-94f7-bbd6a2469fc7)
![2023-05-26_19 38 24](https://github.com/mcenderdragon/Futurepack-API/assets/33810746/fce7b877-0e57-40eb-8546-d46641f41ea2)
![2023-05-26_19 39 07](https://github.com/mcenderdragon/Futurepack-API/assets/33810746/460f4d31-0c24-4d4b-b834-05a36176fe65)
![2023-05-26_19 39 13](https://github.com/mcenderdragon/Futurepack-API/assets/33810746/c119bf48-b831-4570-b7ca-52b2e470bd40)
![2023-05-26_19 39 19](https://github.com/mcenderdragon/Futurepack-API/assets/33810746/5eadae42-b189-41be-bad6-9fca09375fa0)

Minecraft Version: 1.18.2
Forge Version: 40.2.0
Futurepack Version: 1.18.2-33.0.7482

**Comment by mcenderdragon** [comment]
Try the room analyzer it shows how the air is moving. Also 1 lock has enough air for 30 min, so a big room will several hours of air will still just mention its filled for a few %. And keep in my air has gravity so depeding on the plkaned it migfht be jsut all on the floor/ceiling.

**Comment by Dinerell** [comment]
Just to clarify. Wherever I analysed with the E-Scanner it first said everywhere the same percentage and time after that it dropped down again to 0 on any value at any block. Then I used the oxygen tank which got filled up it just changed over on any block with several different values but it always drops everywhere down to 0 again. I kind of used the room analyser but wasn't sure it actually was functional perhaps I just overlooked the messages or something else. By the way the "planet" was the asteroid belt from your mod, so not really sure how much the gravity affects it there. Well, I'm producing oxygen faster than the life support system could transfer it into the room and it is still somehow isn't enough, like if it would leak out.
For now I just overwrote the value for the athmosphere (didn't seem to do anything bad) but as you are already answering I'm probably trying that out.

By the way, I'm just right before that version with the room analyzer and athmosphere fix:
"VersionBot : Version Update 33.0.7499
MCenderdragon : bugfix room analyzer did not render
MCenderdragon : bugfix atmosphere did not work anymore (did not spread air)"

I was under the impression I already had the newest version as I used curseforge literally only a few days ago. I updated and trying it now. Well.. that fixed it.
![2023-05-28_15 57 38](https://github.com/mcenderdragon/Futurepack-API/assets/33810746/149b4808-a7f4-4c34-98ee-c8e1461827de)
